src/etc/inc/web/backup.inc

```diff
@@ -158,7 +166,7 @@ function check_and_returnif_section_exists($section) {
 }
 
 function execPost($post, $postfiles, $ui = true) {
-	global $config, $g;
+	global $config, $g, $backuppath, $savemsg;
 
```

Comment:
Consider revising this function to return:

`array("input_errors" => $input_errors, "savemsg" => $savemsg)`

Again, that would avoid a messy global.

Reply:
Ok! Fixed